lite-rpc/src/bridge.rs

```diff
@@ -107,19 +110,22 @@ impl LiteRpcServer for LiteBridge {
         _start_slot: Slot,
         _config: Option<RpcBlocksConfigWrapper>,
         _commitment: Option<CommitmentConfig>,
-    ) -> crate::rpc::Result<Vec<Slot>> {
+    ) -> RpcResult<Vec<Slot>> {
+        log::info!("Get Blocks");
```

Comment:
should state more clearly that it's not implemented -> "Get Blocks - not implemented")